SPECS/python-virtualenv/python-virtualenv.spec

```diff
@@ -25,6 +25,9 @@ BuildRequires:  python3-pip
 %endif
 
 Requires:       python3
+Requires:       python3-filelock
+Requires:       python3-platformdirs = 2.0.0
```

Comment:
Are you sure you want to set these as =
We should only have the one version in mariner.  And if we upgrade the version in mariner, it's alsmost always for a CVE reason.  So this explicit assignment drags in older CVE riddled packages.

Reply:
Jon, these had upper bound on the version so if we update these runtime packages outside of that upper bound, it might not work so I had this explicit version with the dep. 
![image](https://user-images.githubusercontent.com/106988478/209031741-d01b4e8e-1626-4ff8-b868-4bd963efd09e.png)

Reply:
It still looks incorrect, though, according to your screenshot.
```suggestion
Requires:       python3-platformdirs < 3
```

Reply:
@PawelWMS My bad I screenshotted the latest virtualenv instead of the version we had. 
![image](https://user-images.githubusercontent.com/106988478/209192855-f4b5e5aa-157b-40b2-a3ee-2f84d217fc2f.png)